drivers/ISOSR.py

```diff
@@ -242,13 +242,14 @@ def attach(self, sr_uuid):
 
         self.credentials = os.path.join("/tmp", util.gen_uuid())
         if self.dconf.has_key('type'):
+            # Creation via XC
```

Comment:
That's how we receive it from XAPI. There is an additional type dconf when the request comes in from XC